libsplinter/src/biome/mod.rs

```diff
@@ -32,6 +32,8 @@ pub mod notifications;
 
 #[cfg(feature = "rest-api")]
 pub mod rest_api;
+
+pub mod database;
```

Comment:
This should probably be guarded by `#[cfg(feature="postgres")]` (and an use of `any` once we add more db targets).

It also should be above rest_api for alphabetical sorting

Reply:
All the components within database are already feature guarded. Currently database contains `models`, `schemas`, and `run_postgres_migrations`. The individual structs in `models` and `schemas` are feature guarded, and `run_postgres_migrations` is guarded under the postgres feature. Because of this I don't think database itself needs a guard or if it does have a guard it shouldn't be postgres since it contains components used by other biome modules.

Reply:
Is database the right name for this module then?

Reply:
I think so. It contains the schemas, models, and migrations for biome databases, so it seems appropriate to me, but I'm open to suggestions.

Reply:
It should probably match the other biome features:

`biome::storage`
`biome::user::storage`
etc. 

This might make sense as diesel is a cross cutting concern, and should be enabled with the postgres flag.